woocommerce/woocommerce
# CYS - Fix spacing in patterns

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:







### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new page or post.
2. Insert the `Four Image Grid Content Left` pattern.
3. Check the block spacing between the columns is 4.
<img width="1146" alt="Screenshot 2024-07-18 at 12 22 46" src="https://github.com/user-attachments/assets/05482954-6bb8-4a12-b0c9-0aacc0e9e9ad">

4. Insert the following patterns and check they all have a new spacer block between the title and the rest of the pattern:
- Product Collection 4 Columns
- Product Collection 5 Columns
- Product Collection: Featured Products 5 Columns
- Product Gallery
- Social: Follow us on social media
- Testimonials 3 Columns

5. Insert the `Testimonials single` and the `Testimonials 3 Columns` and check the names of the customers don't have any symbols before them.
6. Insert the `Hero Product Chessboard` and make sure the two picture angles are touching.
<img width="308" alt="Screenshot 2024-07-18 at 14 02 30" src="https://github.com/user-attachments/assets/36e75fa0-154c-4385-8f28-a504cc799b20">



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
CYS - Fix the column spacing for the "Four Image Grid Content Left" pattern
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
